Source/Cells/Spacer/FORMSpacerFieldCell.m

```diff
@@ -0,0 +1,33 @@
+#import "FORMSpacerFieldCell.h"
+@import Hex;
+
+@interface FORMSpacerFieldCell ()
+
+@end
+
+@implementation FORMSpacerFieldCell
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (!self) return nil;
+
+    self.headingLabel.hidden = YES;
+
+    return self;
+}
+
+#pragma mark - FORMBaseFormFieldCell
+
+- (void)updateWithField:(FORMField *)field {
+    [super updateWithField:field];
+
+    self.headingLabel.hidden = YES;
+}
+
+#pragma mark - Layout
+
+- (void)layoutSubviews {
+    [super layoutSubviews];
```

Comment:
Are you referring to the `(void)updateWithField:(FORMField *)field` method or `(void)layoutSubviews`?

Your comment seemed to highlight everything.